app/src/main/java/com/example/kotlinpps/database/DbQuery/DbFetchUserData.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.kotlinpps.database.DbQuery
+
+import android.os.AsyncTask
+import com.example.kotlinpps.ZERO
+import com.example.kotlinpps.database.User
+import com.example.kotlinpps.database.UserDao
+import java.util.concurrent.ExecutionException
+
+
+class DbFetchUserData(private val userDao: UserDao) : AsyncTask<Int, Void, User>(), DbGenericQuery<User, Int> {
+
+    override fun doInBackground(vararg params: Int?): User {
+        return userDao.fetchUserData(params[ZERO]!!)
+    }
+
+    override fun executeQuery(id: Int?): User? {
+        try {
+            return this.execute(id).get()
+        } catch (e: ExecutionException) {
```

Comment:
Merge the catches in one block
catch (FirstException | SecondException | ThirdException e)

Reply:
Kotlin does not support multi exception catch blocks